joomla-projects/privacy-framework
# [Feature] Admin Notifications

Pull Request for Issue #72

### Summary of Changes

Adds notifications in the backend via com_messages when a user creates or confirms their information requests.  Also adds a helper method to `MessagesModelMessage` to send a message to all super users.

### Testing Instructions

Create an information request from the frontend, and confirm an information request created via either the backend or frontend.  The frontend submission and the confirmation should add messages to the message component.
